Automattic/wp-calypso
# Activity Log: Prevent rewinding after page refresh when a rewind process is already running

Currently, users are able to start a rewind, refresh the page, and start another. This results in false positive rewind failures, and could also have adverse effects on the site in certain scenarios.

This PR adds a new selector `isRewinding` which will return a boolean `true` if any rewind is already running. We can pluck this from `state.rewind[siteId].rewind` so that we don't need any additional network expense.

**Testing Instructions**

1. Start a rewind from the activity log.
2. Reload the page.
3. Ensure that the rewind buttons are disabled for the duration of the existing rewind runtime.
4. Ensure that the rewind buttons are enabled after the rewind has completed.
